ReadMe.md

```diff
@@ -3,31 +3,47 @@
 > 1.3.9
 
 ## Installation
-Install all dependencies listed under `package.json`
+Install all NPM dependencies listed under `package.json`
 ```sh
 $ npm install
 ```
 
-## Development Mode
+## Running the Application
+### Development Mode
 We use Webpack to fire off a hot-reloading development server. This allows for continuous code changes without the need to refresh your browser.
 
 ```sh
 $ npm start // Starts localhost:3001 defaulting to APP_ENV=development
 ```
 
-You can also set the APP_ENV variable which dictates what API environment to use as the main source.
+You can also set the `APP_ENV` variable which dictates what API environment to use as the main source.
 ```sh
 $ APP_ENV=development|qa|production npm start // Starts localhost:3001 with set APP_ENV
 ```
 
-## Production Mode
-We use Webpack to fire off a hot-reloading development server. This allows for continuous code changes without the need to refresh your browser.
+### Production Mode
+To replicate the application in a production state. We execute 2 npm scripts with the proper `ENV` variables. By using `NODE_ENV=production`, we disable the hot-reload server. In addition, the `production` Webpack configuration by is set by running `npm run dist`.
+
+* **Step 1**: Build the distribution assets into `./dist/`
+```sh
+$ npm run dist
+```
 
+* **Step 2**: Starts the Node/Express server in `localhost:3001` with the `APP_ENV` set to `production`, targeting the proper `production` API's.
 ```sh
-$ npm run dist // Builds dist path & files
-$ APP_ENV=production NODE_ENV=production npm start // Starts localhost:3001 with set APP_ENV
+$ APP_ENV=production NODE_ENV=production npm start
 ```
 
+## GIT Workflow
+We follow a [feature-branch](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) workflow. If you need to introduce/update the application code, you `SHOULD`:
+
+* Create a new branch off the `development` branch
+* Send a PR pointing to the `development` branch upon completion
+* Once the PR is approved, it should be merged into the `development` branch
+* If there are several PR's in process, a release should be scheduled by merging all completed PR's into the `development` branch
```

Comment:
PRs? If in plural?

Reply:
Hmm yeah, plural. He meant `PRs` and not `PR's`.